config/jobs/kubernetes-sigs/slack-infra/slack-infra-config.yaml

```diff
@@ -7,7 +7,8 @@ presubmits:
     always_run: true
     spec:
       containers:
-        - image: public.ecr.aws/docker/library/golang:1.11.5
+        - image: gcr.io/k8s-staging-releng/releng-ci:latest-go1.23-bookworm
```

Comment:
let's not increase our exposure to staging images, is there a problem with the ECR image?

Reply:
(I mean sure 1.11 doesn't exist anymore, but we can use a current tag? 1.11 is wildly old now)

Reply:
I can update to use ecr

Reply:
Let's do that, everywhere we pull from staging repos is going to be a headache.

Reply:
there is no golang:1.11 in ecr the last min is go1.14, only in docker hub

Reply:
but why not bump to the latest go? if there are things to fix i will follow up

Reply:
If we bump to latest go we can still use a different version at any time in the job because of https://go.dev/doc/toolchain

it will bootstrap itself to the right version, we can control it in the go.mod